Source/WebInspectorUI/UserInterface/Views/DOMTreeElement.js

```diff
@@ -2033,14 +2033,19 @@ WI.DOMTreeElement = class DOMTreeElement extends WI.TreeElement
             text = WI.unlocalizedString("grid");
             handleClick = this._layoutBadgeClicked.bind(this);
             break;
+
+        case WI.DOMTreeElement.BadgeType.Event:
+            text = WI.UIString("Event");
```

Comment:
this is using an existing UI string, which is why it's not in the localization file (and why i didn't add a comment for it, since we'd already have a translation for it)